packages/pdf-generator/src/components/PDFDocument.js

```diff
@@ -7,7 +7,8 @@ import {
     Document
 } from '@react-pdf/renderer';
 import RHLogo from './Logo';
-import { DateFormat } from '@redhat-cloud-services/frontend-components/DateFormat';
+// eslint-disable-next-line rulesdir/disallow-fec-relative-imports
+import { DateFormat } from '@redhat-cloud-services/frontend-components';
```

Comment:
Did some build errors occur?

Reply:
Yes there are errors with the UMD build on node v14.16.0:
> @redhat-cloud-services/frontend-components-pdf-generator: Error: 'DateFormat' is not exported by ../components/DateFormat/index.js, imported by src/components/PDFDocument.js